app/src/test/java/org/simple/clinic/registration/confirmpin/RegistrationConfirmPinScreenControllerTest.kt

```diff
@@ -47,7 +46,6 @@ class RegistrationConfirmPinScreenControllerTest {
   @Test
   fun `when 4 digits are entered then the PIN should be submitted automatically`() {
     whenever(userSession.ongoingRegistrationEntry()).thenReturn(Single.just(OngoingRegistrationEntry(pin = "1234")))
-    whenever(userSession.saveOngoingRegistrationEntry(any())).thenReturn(Completable.complete())
```

Comment:
Should we add `verify` statements to verify that method is being called instead?

Reply:
Good catch, I'll add it!

Reply:
I just realised that most of the tests already had the assertion, it just was not visible in the diff on GitHub. One test was missing it, so I added it.